application/build.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
     implementation 'org.jetbrains:annotations:23.0.0'
 
     implementation project(':database')
+    implementation project(':inmemorycompiler')
```

Comment:
I know its annoying that this comes now, but I am actually against having a dedicated subproject.

Last year kinda showed that everything in its own project gives annoyance since its so "far away" from the main project. It starts to rot and adding utility that has to be used across both can be hard to get right.

For example, you are missing the non-null-by-default and some stuff, like spotless utility, has to be explicitly applied correctly to hit those projects as well.

All in all, i think its easier if we move this into the main project, with a dedicated subpackage.

Reply:
actually wait. this is a separate "project" thats used by the command, right? then its okay to have it external. but it should still use nonnull features etc.

Reply:
yes, it's basically like an entire separate project